docs/book/src/component-config-tutorial/testdata/project/test/e2e/e2e_test.go

```diff
@@ -0,0 +1,77 @@
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"tutorial.kubebuilder.io/project/test/utils"
+)
+
+const namespace = "controller-system"
+
+var _ = Describe("controller", Ordered, func() {
+	Context("Operator", func() {
+		It("should run successfully", func() {
+			var controllerPodName string
+			var err error
+
+			// operatorImage stores the name of the image used in the example
+			var operatorImage = "controller:latest"
```

Comment:
```suggestion
			var projectimage = "projectname:latest"
```

I think we should change here ^